airflow/models.py

```diff
@@ -366,8 +365,8 @@ def kill_zombies(self, session=None):
                 if ti.task_id in dag.task_ids:
                     task = dag.get_task(ti.task_id)
                     ti.task = task
-                    ti.handle_failure("%s killed as zombie", ti)
-                    self.logger.info('Marked zombie job %s as failed', ti)
+                    ti.handle_failure("%s killed as zombie".format(ti))
```

Comment:
You want `{}` if you use `format`

Reply:
This is fixed in the latest commit, thanks